public/index.html

```diff
@@ -4,85 +4,12 @@
     <meta charset="utf-8" />
```

Comment:
As an aside, I'm sort of surprised that this /index.html page works.  I would have thought the "**" rewrite to the "api" function would have taken precedence, but I guess it does work, so... cool.

Reply:
huh... yeah, it seems like it should get rewritten, but I guess not 🤷